### ISSUE TITLE
ADBungFu just deleted all my apps?

### ISSUE THREAD

**Issue by MosaKasem** [MuntashirAkon/AppManager#1425]
### Please check before submitting an issue

- [X] I know what my device, OS and App Manager versions are
- [X] I know how to take logs
- [ ] I know how to reproduce the issue which may not be specific to my device

### Describe the bug

Everything is gone, my stock apps too, what the actual fuck?
I thought the app was meant for clearing or reducing the cached data stored by applications on an Android device??

### To Reproduce

"select action"
"Trim caches for all apps"
"Perform idle maintenance now"

### Expected behavior

To clear those cached background apps by Android

### Logs

"MatLog" is blank now after running the operation, it's not even producing/output anything onto the log now.

### Device info

- Device: Poco X5 Pro 5G
- OS Version: Android 14 (crDroid 10.1)
- App Manager Version: ?
- Mode: Root

### Additional context

Basically, opened the app, selected the filter "stand-by apps" disallowed apps that serve no purpose for me to run in the background. Hit the middle button that has an arrow pointing down (the command execute menu), hit "Trim caches for all apps", then "perform idle maintenance now". Then after that, all those apps I "disallowed" were gone.
Checked apps under settings, nothing there either.

**Comment by MosaKasem** [comment]
Am sorry, I rushed to first link without checking, f""" my life. Wrong forum.
